### ISSUE TITLE
Attack for multi-label classification

### ISSUE THREAD

**Issue by danush-95** [bethgelab/foolbox#643]
Is there an attack available for multi-label classification problems? I went through the repo and I could see the ```self.run``` ends with ```ep.crossentropy```. However, ```BCE with logits``` is required for the above-stated problem. Is there an attack or method which can be over-written to implement this loss?
